README.md

```diff
@@ -120,131 +168,156 @@ Pre-requisites
     ```
 
     </details>
-- if you are not on the most recent version of OSX, `homebrew`
-    - this allows us to install the current version of cocoapods without
-      running into ruby incompatibilities - e.g.
-      https://github.com/CocoaPods/CocoaPods/issues/11763
 
-Important
----
-Most of the recent issues encountered have been due to incompatible setup. We
-have now:
-- locked down the dependencies,
-- created setup and teardown scripts to setup self-contained environments with
-  those dependencies, and
-- CI enabled to validate that they continue work.
 
-If you have setup failures, please compare the configuration in the passing CI
-builds with your configuration. That is almost certainly the source of the error.
+__2. Installing (one time only)__
 
-Installing (one time only)
----
-Run the setup script for the platform you want to build
+- Run the setup script for the platform you want to build
 
 ```
-$ bash setup/setup_android_native.sh
+bash setup/setup_android_native.sh
 AND/OR
-$ bash setup/setup_ios_native.sh
+bash setup/setup_ios_native.sh
 ```
 
 **(optional)** Configure by changing the files in `www/json`.
 Defaults are in `www/json/*.sample`
 
 ```
-$ ls www/json/*.sample
-$ cp www/json/startupConfig.json.sample www/json/startupConfig.json
-$ cp ..... www/json/connectionConfig.json
+ls www/json/*.sample
+cp www/json/startupConfig.json.sample www/json/startupConfig.json
+cp ..... www/json/connectionConfig.json
 ```
 
-### Activation (after install, and in every new shell)
-
-```
-$ source setup/activate_native.sh
-```
-
-### Activation (after install, and in every new shell)
-
 If connecting to a development server over http, make sure to turn on http support on android
 
 ```
     <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application">
         <application android:usesCleartextTraffic="true"/>
     </edit-config>
 ```
+__3. Run this in every new shell__ 
+
+- __Activation__
+```
+source setup/activate_native.sh
+```
+<details><summary> Expected Output </summary>
+
+``` 
+Activating nvm
+Using version 19.5.0
+Now using node v19.5.0 (npm v9.3.1)
+npm version = 9.3.1
+Adding cocoapods to the path
+Verifying /Users/<username>/Library/Android/sk or /Users/<username>/Library/Android/sdk is set
+Activating sdkman, and by default, gradle
+Ensuring that we use the most recent version of the command line tools
+Configuring the repo for building native code
+Copied config.cordovabuild.xml -> config.xml and package.cordovabuild.json -> package.json
+```
+
+</details>
 
-### Run in the emulator
+- __Pick a type of build and execute the following:__
```

Comment:
And you don't need to build as part of activation and you don't need to activate before every build.